components/docs-chef-io/content/automate/restore.md

```diff
@@ -52,6 +52,14 @@ Ensure access for the backup type used:
          # """
     ```
 
+1. To restore a backup to a machine with less memory than the original system, adjust for the appropriate lower memory settings by creating a `patch.toml` file that specifies the heapsize, and providing the file at restore time:
```

Comment:
We might want to say something about "creating a patch.toml or adding to an existing one", lest someone think they must (or can) submit a toml file for every config change they want to make prior to restore.